<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> The O'Brien Lab </title> <meta name="author" content="The O'Brien Lab "> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://obrien-lab.github.io/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">About <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog </a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications </a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/books/">bookshelf</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/blog/">blog</a> </div> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">The O'Brien Lab</span> </h1> <p class="desc"></p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/prof_pic_ed-480.webp 480w,/assets/img/prof_pic_ed-800.webp 800w,/assets/img/prof_pic_ed-1400.webp 1400w," type="image/webp" sizes="(min-width: 930px) 270.0px, (min-width: 576px) 30vw, 95vw"> <img src="/assets/img/prof_pic_ed.jpg?e396b44327bb78328cedc45d296ad26c" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="prof_pic_ed.jpg" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </source></picture> </figure> <div class="more-info"> <p>XXX Benkovic Building</p> <p>Penn State University</p> <p>University Park</p> </div> </div> <div class="clearfix"> <p>Developing and applying Physical Bioinformatic techniques to measure rates of translation transcriptome-wide and their molecular origins as relates to fundamental biology and disease.</p> </div> <h2> <a href="/news/" style="color: inherit">news</a> </h2> <div class="news"> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row" style="width: 20%">Mar 14, 2025</th> <td> <a class="news-title" href="/news/announcement_1/">Protein accidentally lassos itself, helping explain unusual refolding behavior (Sci. Adv paper)</a> </td> </tr> <tr> <th scope="row" style="width: 20%">Feb 12, 2025</th> <td> Congrats Dr. Ian Sitarik has successfully defended his thesis titled “Understanding the fundamental principles behind the influences of native and non-native entanglements on protein structure, function, and misfolding” </td> </tr> <tr> <th scope="row" style="width: 20%">Nov 07, 2015</th> <td> <a class="news-title" href="/news/announcement_2/">A long announcement with details</a> </td> </tr> </table> </div> </div> <h2> <a href="/blog/" style="color: inherit">latest posts</a> </h2> <div class="news"> <div class="table-responsive" style="max-height: 60vw"> <table class="table table-sm table-borderless"> <tr> <th scope="row" style="width: 20%">Mar 26, 2025</th> <td> <a class="news-title" href="/blog/2025/plotly/">a post with plotly.js</a> </td> </tr> <tr> <th scope="row" style="width: 20%">Dec 04, 2024</th> <td> <a class="news-title" href="/blog/2024/photo-gallery/">a post with image galleries</a> </td> </tr> <tr> <th scope="row" style="width: 20%">May 14, 2024</th> <td> <a class="news-title" href="https://blog.google/technology/ai/google-gemini-update-flash-ai-assistant-io-2024/" target="_blank" rel="external nofollow noopener">Google Gemini updates: Flash 1.5, Gemma 2 and Project Astra</a> <svg width="2rem" height="2rem" viewbox="0 0 40 40" xmlns="http://www.w3.org/2000/svg"> <path d="M17 13.5v6H5v-12h6m3-3h6v6m0-6-9 9" class="icon_svg-stroke" stroke="#999" stroke-width="1.5" fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round"></path> </svg> </td> </tr> </table> </div> </div> <h2> <a href="/publications/" style="color: inherit">selected publications</a> </h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Samelson2018" class="col-sm-8"> <div class="title">Kinetic and structural comparison of a protein’s cotranslational folding and refolding pathways</div> <div class="author"> Avi J. Samelson, Eric Bolin, Shawn M. Costello, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Ajeet K. Sharma, Edward P. O’Brien, Susan Marqusee' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> 2018 </div> <div class="periodical"> Cited by: 43; All Open Access, Gold Open Access, Green Open Access </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1126/sciadv.aas9098" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Precise protein folding is essential for the survival of all cells, and protein misfolding causes a number of diseases that lack effective therapies, yet the general principles governing protein folding in the cell remain poorly understood. In vivo, folding can begin cotranslationally and protein quality control at the ribosome is essential for cellular proteostasis. We directly characterize and compare the refolding and cotranslational folding trajectories of the protein HaloTag. We introduce new techniques for both measuring folding kinetics and detecting the conformations of partially folded intermediates during translation in real time. We find that, although translation does not affect the rate-limiting step of HaloTag folding, a key aggregation-prone intermediate observed during in vitro refolding experiments is no longer detectable. This rerouting of the folding pathway increases HaloTag’s folding efficiency and may serve as a general chaperone-independent mechanism of quality control by the ribosome. © 2018 The Authors.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Sharma201894" class="col-sm-8"> <div class="title">Non-equilibrium coupling of protein structure and function to translation–elongation kinetics</div> <div class="author"> Ajeet K Sharma and Edward P O’Brien </div> <div class="periodical"> 2018 </div> <div class="periodical"> Cited by: 42; All Open Access, Bronze Open Access </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.sbi.2018.01.005" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Protein folding research has been dominated by the assumption that thermodynamics determines protein structure and function. And that when the folding process is compromised in vivo the proteostasis machinery — chaperones, deaggregases, the proteasome — work to restore proteins to their soluble, functional form or degrade them to maintain the cellular pool of proteins in a quasi-equilibrium state. During the past decade, however, more and more proteins have been identified for which altering only their speed of synthesis alters their structure and function, the efficiency of the down-stream processes they take part in, and cellular phenotype. Indeed, evidence has emerged that evolutionary selection pressures have encoded translation-rate information into mRNA molecules to coordinate diverse co-translational processes. Thus, non-equilibrium physics can play a fundamental role in influencing nascent protein behavior, mRNA sequence evolution, and disease. Here, we discuss how our understanding of this phenomenon is being advanced by the application of theoretical tools from the physical sciences. © 2018 Elsevier Ltd</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Fritch20185077" class="col-sm-8"> <div class="title">Origins of the Mechanochemical Coupling of Peptide Bond Formation to Protein Synthesis</div> <div class="author"> Benjamin Fritch, Andrey Kosolapov, Phillip Hudson, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Daniel A. Nissley, H. Lee Woodcock, Carol Deutsch, Edward P. O’Brien' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> 2018 </div> <div class="periodical"> Cited by: 32; All Open Access, Green Open Access </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/jacs.7b11044" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Mechanical forces acting on the ribosome can alter the speed of protein synthesis, indicating that mechanochemistry can contribute to translation control of gene expression. The naturally occurring sources of these mechanical forces, the mechanism by which they are transmitted 10 nm to the ribosome’s catalytic core, and how they influence peptide bond formation rates are largely unknown. Here, we identify a new source of mechanical force acting on the ribosome by using in situ experimental measurements of changes in nascent-chain extension in the exit tunnel in conjunction with all-atom and coarse-grained computer simulations. We demonstrate that when the number of residues composing a nascent chain increases, its unstructured segments outside the ribosome exit tunnel generate piconewtons of force that are fully transmitted to the ribosome’s P-site. The route of force transmission is shown to be through the nascent polypetide’s backbone, not through the wall of the ribosome’s exit tunnel. Utilizing quantum mechanical calculations we find that a consequence of such a pulling force is to decrease the transition state free energy barrier to peptide bond formation, indicating that the elongation of a nascent chain can accelerate translation. Since nascent protein segments can start out as largely unfolded structural ensembles, these results suggest a pulling force is present during protein synthesis that can modulate translation speed. The mechanism of force transmission we have identified and its consequences for peptide bond formation should be relevant regardless of the source of the pulling force. © 2018 American Chemical Society.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Sharma20176775" class="col-sm-8"> <div class="title">Increasing Protein Production Rates Can Decrease the Rate at Which Functional Protein is Produced and Their Steady-State Levels</div> <div class="author"> Ajeet K. Sharma and Edward. P. O’Brien </div> <div class="periodical"> 2017 </div> <div class="periodical"> Cited by: 11 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/acs.jpcb.7b01700" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The rate at which soluble, functional protein is produced by the ribosome has recently been found to vary in complex and unexplained ways as various translation-associated rates are altered through synonymous codon substitutions. To understand this phenomenon, here, we combine a well-established ribosome-traffic model with a master-equation model of cotranslational domain folding to explore the scenarios that are possible for the protein production rate, J, and the functional-nascent protein production rate, F, as the rates of various translation processes are altered for five different E. coli proteins. We find that while J monotonically increases as the rates of translation-initiation, -elongation, and -termination increase, F can either increase or decrease. We show that F’s nonmonotonic behavior arises within the model from two opposing trends: the tendency for increased translation rates to produce more total protein but less cotranslationally folded protein. We further demonstrate that under certain conditions these nonmonotonic changes in F can result in nonmonotonic variations in post-translational, steady-state levels of functional protein. These results provide a potential explanation for recent experimental observations in which the specific activity of enzymatic proteins decreased with increased synthesis rates. Additionally our model has the potential to be used to rationally design transcripts to maximize the production of functional nascent protein by simultaneously optimizing translation initiation, elongation, and termination rates. © 2017 American Chemical Society.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Sharma2018" class="col-sm-8"> <div class="title">Determinants of translation speed are randomly distributed across transcripts resulting in a universal scaling of protein synthesis times</div> <div class="author"> Ajeet K. Sharma, Nabeel Ahmed, and Edward P. O’Brien </div> <div class="periodical"> 2018 </div> <div class="periodical"> Cited by: 18; All Open Access, Bronze Open Access </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1103/PhysRevE.97.022409" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>Ribosome profiling experiments have found greater than 100-fold variation in ribosome density along mRNA transcripts, indicating that individual codon elongation rates can vary to a similar degree. This wide range of elongation times, coupled with differences in codon usage between transcripts, suggests that the average codon translation-rate per gene can vary widely. Yet, ribosome run-off experiments have found that the average codon translation rate for different groups of transcripts in mouse stem cells is constant at 5.6 AA/s. How these seemingly contradictory results can be reconciled is the focus of this study. Here, we combine knowledge of the molecular factors shown to influence translation speed with genomic information from Escherichia coli, Saccharomyces cerevisiae and Homo sapiens to simulate the synthesis of cytosolic proteins in these organisms. The model recapitulates a near constant average translation rate, which we demonstrate arises because the molecular determinants of translation speed are distributed nearly randomly amongst most of the transcripts. Consequently, codon translation rates are also randomly distributed and fast-translating segments of a transcript are likely to be offset by equally probable slow-translating segments, resulting in similar average elongation rates for most transcripts. We also show that the codon usage bias does not significantly affect the near random distribution of codon translation rates because only about 10% of the total transcripts in an organism have high codon usage bias while the rest have little to no bias. Analysis of Ribo-Seq data and an in vivo fluorescent assay supports these conclusions. © 2018 American Physical Society.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Leininger20195523" class="col-sm-8"> <div class="title">Domain topology, stability, and translation speed determine mechanical force generation on the ribosome</div> <div class="author"> Sarah E. Leininger, Fabio Trovato, Daniel A. Nissley, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Edward P. O’Brien' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> 2019 </div> <div class="periodical"> Cited by: 38 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1073/pnas.1813003116" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>The concomitant folding of a nascent protein domain with its synthesis can generate mechanical forces that act on the ribosome and alter translation speed. Such changes in speed can affect the structure and function of the newly synthesized protein as well as cellular phenotype. The domain properties that govern force generation have yet to be identified and understood, and the influence of translation speed is unknown because all reported measurements have been carried out on arrested ribosomes. Here, using coarse-grained molecular simulations and statistical mechanical modeling of protein synthesis, we demonstrate that force generation is determined by a domain’s stability and topology, as well as translation speed. The statistical mechanical models we create predict how force profiles depend on these properties. These results indicate that force measurements on arrested ribosomes will not always accurately reflect what happens in a cell, especially for slow-folding domains, and suggest the possibility that certain domain properties may be enriched or depleted across the structural proteome of organisms through evolutionary selection pressures to modulate protein synthesis speed and posttranslational protein behavior. © 2019 National Academy of Sciences. All Rights Reserved.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="O'Brien20077346" class="col-sm-8"> <div class="title">Interactions between hydrophobic and ionic solutes in aqueous guanidinium chloride and urea solutions: Lessons for protein denaturation mechanism</div> <div class="author"> Edward P. O’Brien, Ruxandra I. Dima, Bernard Brooks, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'D. Thirumalai' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> 2007 </div> <div class="periodical"> Cited by: 336 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/ja069232+" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>In order to clarify the mechanism of denaturant-induced unfolding of proteins we have calculated the interactions between hydrophobic and ionic species in aqueous guanidinium chloride and urea solutions using molecular dynamics simulations. Hydrophobic association is not significantly changed in urea or guanidinium chloride solutions. The strength of interaction between ion pairs is greatly diminished by the guanidinium ion. Although the changes in electrostatic interactions in urea are small, examination of structures, using appropriate pair functions, of urea and water around the solutes show strong hydrogen bonding between urea’s carbonyl oxygen and the positively charged solute. Our results strongly suggest protein denaturation occurs by the direct interaction model according to which the most commonly used denaturants unfold proteins by altering electrostatic interactions either by solvating the charged residues or by engaging in hydrogen bonds with the protein backbone. To further validate the direct interaction model we show that, in urea and guanidinium chloride solutions, unfolding of an unusually stable helix (H1) from mouse PrPC (residues 144-153) occurs by hydrogen bonding of denaturants to charged side chains and backbone carbonyl groups. © 2007 American Chemical Society.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="Sharma20161180" class="col-sm-8"> <div class="title">Physical Origins of Codon Positions That Strongly Influence Cotranslational Folding: A Framework for Controlling Nascent-Protein Folding</div> <div class="author"> Ajeet K. Sharma, Bernd Bukau, and Edward P. O’Brien </div> <div class="periodical"> 2016 </div> <div class="periodical"> Cited by: 27 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1021/jacs.5b08145" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">DOI</a> </div> <div class="abstract hidden"> <p>An emerging paradigm in the field of in vivo protein biophysics is that nascent-protein behavior is a type of nonequilibrium phenomenon, where translation-elongation kinetics can be more important in determining nascent-protein behavior than the thermodynamic properties of the protein. Synonymous codon substitutions, which change the translation rate at select codon positions along a transcript, have been shown to alter cotranslational protein folding, suggesting that evolution may have shaped synonymous codon usage in the genomes of organisms in part to increase the amount of folded and functional nascent protein. Here, we develop a Monte Carlo-master-equation method that allows for the control of nascent-chain folding during translation through the rational design of mRNA sequences to guide the cotranslational folding process. We test this framework using coarse-grained molecular dynamics simulations and find it provides optimal mRNA sequences to control the simulated, cotranslational folding of a protein in a user-prescribed manner. With this approach we discover that some codon positions in a transcript can have a much greater impact on nascent-protein folding than others because they tend to be positions where the nascent chain populates states that are far from equilibrium, as well as being dependent on a complex ratio of time scales. As a consequence, different cotranslational profiles of the same protein can have different critical codon positions and different numbers of synonymous mRNA sequences that encode for them. These findings explain that there is a fundamental connection between the nonequilibrium nature of cotranslational processes, nascent-protein behavior, and synonymous codon usage. © 2015 American Chemical Society.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2025 The O'Brien Lab . Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>